src/appleseed/foundation/image/colorspace.h

```diff
@@ -236,10 +237,19 @@ Color<T, 3> linear_rgb_to_hsl(const Color<T, 3>& linear_rgb);
 template <typename T>
 Color<T, 3> ciexyz_to_linear_rgb(const Color<T, 3>& xyz);
 
+template <typename T>
+Color<T, 3> ciexyz_to_linear_rgb(
```

Comment:
Ok, so in this case we have to use XYZ. The rest is fine. The "linear" is really redundant since the XYZ<>RGB transformation has to be done with linear values, but i'm totally ok with leaving it.